iOS/ExposureNotification/cgmanifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Registrations":[ 
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "repositoryUrl": "https://github.com/xamarin/XamarinComponents/tree/master/iOS/ExposureNotification",
```

Comment:
This is meant to be the original native repo, so I am not sure this file has to be here if it is an OS API.

Reply:
Should I remove it?

Reply:
Oh right! I think yes, since the framework will live in users macs, unless you want to point to Xcode download link. 😛

Reply:
Deleted it.